packages/core/src/main/Account.ts

```diff
@@ -260,8 +262,10 @@ class Account extends EventEmitter {
 
     switch (event.type) {
       case CONVERSATION_EVENT.OTR_MESSAGE_ADD: {
-        const decodedMessage = await this.decodeGenericMessage(event as ConversationOtrMessageAddEvent);
-        return {...decodedMessage, from, conversation};
+        const otrMessage = event as ConversationOtrMessageAddEvent;
+        const sessionId = CryptographyService.constructSessionId(from, otrMessage.data.sender);
+        const decodedMessage = await this.decodeGenericMessage(otrMessage);
+        return {...decodedMessage, from, conversation, sessionId};
```

Comment:
Returning `sessionId` is not needed anymore, is it?

Reply:
It is, because it is now part of the contract of `PayloadBundle`.